protocol-swanky/src/main/scala/org/ensime/server/protocol/swank/SwankFormats.scala

```diff
@@ -357,6 +357,7 @@ object SwankProtocolResponse {
   implicit object EnsimeEventFormat extends TraitFormat[EnsimeEvent] {
     def write(ee: EnsimeEvent): Sexp = ee match {
       case e: AnalyzerReadyEvent.type => wrap(e)
+      case e: AsyncConnectionInfo => wrap(e)
```

Comment:
I am not quite sure here. Is it okay that way?

Reply:
yeah I think that looks fine